packages/wallets/src/features/cfd/screens/Jurisdiction/JurisdictionScreen.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { FC, useMemo } from 'react';
+import { useAvailableMT5Accounts } from '@deriv/api';
+import { WalletText } from '../../../../components/Base/WalletText';
+import { useModal } from '../../../../components/ModalProvider';
+import { JurisdictionCard } from './JurisdictionCard';
+import './JurisdictionScreen.scss';
+
+type TJurisdictionScreenProps = {
+    selectedJurisdiction: string;
```

Comment:
@adrienne-deriv maybe would be better to add type of allowed jurisdictions instead of string? Just a suggestion, you can ignore it.

Reply:
` selectedJurisdiction: NonNullable<ReturnType<typeof useAvailableMT5Accounts>['data']>[0]['shortcode'];`

Something like this ?

Reply:
@thisyahlen-deriv I thought it's a good idea, but we have `string` type for this `shortcode` property in api types so I guess we can leave it as string

Reply:
Gotcha, I've updated the types in my next PR: https://github.com/binary-com/deriv-app/pull/10739/commits/44e4875736230c4d995a07f54e4e6e432994d900